airflow/utils/db.py

```diff
@@ -1887,3 +1902,14 @@ def get_query_count(query_stmt: Select, session: Session) -> int:
     """
     count_stmt = select(func.count()).select_from(query_stmt.order_by(None).subquery())
     return session.scalar(count_stmt)
+
+
+def exists_query(*where: ClauseElement, session: Session) -> bool:
```

Comment:
```suggestion
def exists_query(*where: ClauseElement, *, session: Session) -> bool:
```

Reply:
Can’t do this because `where` is already a star args. Bare `*` is (syntactically) a special case of `*name` and you can only have one star argument in the argument list.